kolibri/core/assets/src/vue/progress-bar/index.vue

```diff
@@ -0,0 +1,59 @@
+<template>
+
+  <div>
+    <div class="visuallyhidden" id="progress-bar-label">{{ $tr('yourProgressIs') }}</div>
+    <div class="progress-bar-wrapper"
+      role="progressbar"
+      aria-labelledby="progress-bar-label"
+      :aria-valuenow="progressPercent"
+      aria-valuemin="0"
+      aria-valuemax="100">
+      <div class="progress-bar-complete" v-bind:style="{ width: progressPercent + '%' }"></div>
+    </div>
+    <p class="progress-bar-text">{{ progressPercent + '% ' + $tr('complete') }}</p>
+  </div>
+
+</template>
+
+
+<script>
+
+  module.exports = {
+    $trNameSpace: 'progressBar',
+    $trs: {
+      yourProgressIs: 'Your progress is:',
+      complete: 'Complete',
+    },
+    props: {
+      progressPercent: {
+        type: Number,
+        default: 0,
+        required: true,
```

Comment:
If something is required, it shouldn't need a default.

Also, would be great to add a validator to this like you have on the other component. Right now it's not obvious whether it takes a value in the range [0, 100] or [0.0, 1.0]